pkg/mountutil/volumestore/volumestore.go

```diff
@@ -87,6 +91,19 @@ func (vs *volumeStore) Create(name string) (*native.Volume, error) {
 		if err := os.Mkdir(volDataPath, 0755); err != nil {
 			return err
 		}
+
+		volFilePath := filepath.Join(volPath, volumeFileName)
+		labelsMap := strutil.ConvertKVStringsToMap(labels)
+
+		volumeOpts := make(map[string]interface{})
+		volumeOpts["nerdctlLabels"] = labelsMap
```

Comment:
Just `labels` is fine for volumes, as we own the entire JSON by ourselves

Reply:
And better to define a struct

Reply:
fixed